packages/phpstan-latte-rules/tests/Rules/LatteCompleteCheckRule/LatteCompleteCheckRuleTest.php

```diff
@@ -30,31 +30,39 @@ public function provideData(): Iterator
     {
         // tests @see \PHPStan\Rules\Methods\CallMethodsRule
         $errorMessage = sprintf('Call to an undefined method %s::missingMethod().', SomeTypeWithMethods::class);
-        yield [__DIR__ . '/Fixture/SomeMissingMethodCall.php', [[$errorMessage, 16]]];
+//        yield [__DIR__ . '/Fixture/SomeMissingMethodCall.php', [[$errorMessage, 12]]];
```

Comment:
here I probably lost some context and types of variables because of changing MethodCall to ClassMethod

Reply:
MethodCall must be used, as ClassMethod has no idea about types inside itself.

To find ClassMethod, use node finder and patent attribute

Reply:
that's the problem in presenters, there is no such call, you have usually only template variable assigns, nothing else
```
public function renderDefault(): void
{
    $this->template->foo = 'bar';
}
```

Maybe I can change the rule to abstract Node and process UI\Controls the "old" way and presenters with something else.

Reply:
Rewritten using Node, each "Holder" decide which nodes can handle